packages/cadl-rlc-test/test/parametrizedHost/spec/main.cadl

```diff
@@ -0,0 +1,63 @@
+import "@cadl-lang/rest";
+import "@cadl-lang/versioning";
+import "@cadl-lang/openapi";
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-autorest";
+
+using Cadl.Http;
+using Cadl.Rest;
+using Cadl.Versioning;
+using Azure.Core;
+using OpenAPI;
+
+//
+// ** Service description **
+//
+
+@service({title: "ParametrizedHost"})
+@useAuth(OAuth2Auth<[MyFlow]>)
+@versionedDependency(Azure.Core.Versions.v1_0_Preview_1)
+@server(
+  "{host}.{subdomain}.{sufix}.com/{apiVersion}",
+  "Confidential Ledger Service",
+  {
+    host: "one",
+    subdomain: "two",
+    sufix: "three",
+    apiVersion: "v1"
```

Comment:
I wonder in this case, if these parameters are string type with a default value or just const strings? It looks like we are not treating it as const string here https://github.com/Azure/autorest.typescript/pull/1643/files#diff-26365d8da06b69db64937725fd469585f163940932d720de759bc478d5be1705R20-R23

Reply:
+1 I think the more proper way to indicate default value would be:
```
host?: string = "one";
```